pypowsybl/__init__.py

```diff
@@ -56,3 +56,9 @@ def is_config_read() -> bool:
 
 def print_version() -> None:
     print(_pypowsybl.get_version_table())
+
+def set_loadflow_provider(provider: str) -> None:
```

Comment:
I think we should be more specific by naming method `set_default_` and `get_default_...`.

Also it deserves some docstrings, and to be added in the API reference in the doc :) (loadflow.rst in reference directory)